portfolio/src/main/webapp/script.js

```diff
@@ -53,14 +53,53 @@ function makeUglySite() {
 }
 
 function getComments() {
-  const url = "/data?limit=" + getLimit();
+  const url = '/data?limit=' + getLimit();
   fetch(url).then(response => response.json()).then((comments) => {
-    document.getElementById('comment-container').innerText = comments.join(', ');
+    const commentListElement = document.getElementById('comment-container');
+    commentListElement.innerText = '';
+    comments.forEach((comment) => {
+      const fullComment = createDivElement('', 'full-comment');
+      const commentText = createDivElement(comment, 'comment-class');
+      const replyButton = document.createElement('button');
+      replyButton.textContent = 'Reply';
+      replyButton.classList = 'reply-button';
+      replyButton.addEventListener('click', () => {
+        replyButton.style.display = 'none';
+        showReplyField(fullComment);
+      });
+      fullComment.appendChild(commentText);
+      fullComment.appendChild(replyButton);
+      commentListElement.append(fullComment);
+    })
   });
 }
 
 function getLimit() {
-  const limit = document.getElementById("limit").value;
+  const limit = document.getElementById('limit').value;
   return limit;
 }
 
+function createDivElement(text, className) {
+  const divElement = document.createElement('div');
+  divElement.classList = className;
```

Comment:
Instead of overriding the value like this, I think you can use the method, .classList.add(className)

Reply:
^ yes similar to the example I posted earlier this morning

Reply:
got it